modules/ROOT/pages/deploy-proxy-monitoring.adoc

```diff
@@ -38,13 +38,20 @@ image::{imagesprefix}zdm-ansible-container-ls3.png[Contents of the Ansible Contr
 [[_configure_the_zdm_proxy]]
 == Configure the {zdm-proxy}
 
-The {zdm-proxy} configuration is composed of four files:
+The {zdm-proxy} configuration is composed of five files:
 
+* `zdm_proxy_container_config.yml`, containing internal configuration of proxy container itself.
 * `zdm_proxy_cluster_config.yml`, containing all the configuration that allows the proxy to connect to Origin and Target. This is always required.
 * `zdm_proxy_core_config.yml`, containing important configuration that is commonly used and changed during the migration.
 * `zdm_proxy_advanced_config.yml`, containing advanced configuration that may be required in some scenarios but can be usually left to its default values.
 * `zdm_proxy_custom_tls_config.yml`, to configure TLS encryption if desired.
 
+=== Container configuration
+The first step of the proxy container configuration is to open the `zdm_proxy_container_config.yml` file.
+Configure the desired ZDM proxy version and determine the configuration parameters. 
```

Comment:
I would say here `determine strategy to inject configuration parameters`. You do not configure ZDM proxy parameters here, but the way of providing them (file vs. environment variables).